wp-content/themes/magnat/demo-import/main/theme-options.txt
{"last_tab":"12","favicon-img-default":{"url":"","id":"","height":"","width":"","thumbnail":""},"logo-option-default":"1","logo-img-default":{"url":"http:\/\/magnat.n2mu.studio\/main\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-sticky-default":{"url":"http:\/\/magnat.n2mu.studio\/main\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-mobile-default":{"url":"http:\/\/magnat.n2mu.studio\/main\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-height-default":"28","logo-text-default":"Magnat","page-preloader":"","page-background-default":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"header-layout-default":"header1","header-top-height-default":"50","header-middle-height-default":"100","header-bottom-height-default":"50","header-mobile-height-default":"75","header-mobile-point":"1000","header-tools-search":"1","header-tools-cart":"1","header-sticky":"","footer-columns-default":"4","footer-top-borders-default":"","footer-bottom-enabled-default":"1","copyrights":"Copyright \u00a9 2017 - <a href=\"http:\/\/magnat.n2mu.studio\" target=\"_blank\">Magnat Theme<\/a>","footer-top-spacing-default":{"padding-top":"50px","padding-bottom":"50px"},"footer-bottom-spacing-default":{"padding-top":"10px","padding-bottom":"10px"},"footer-background-default":{"background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"color-main-theme":"#33C6C6","color-buttons":"#33C6C6","color-buttons-font":"#ffffff","color-buttons-hover":"#33C6C6","color-buttons-font-hover":"#efefef","color-body":"#72787b","color-h1":"#2b2b2b","color-h2":"#2b2b2b","color-h3":"#2b2b2b","color-h4":"#2b2b2b","color-h5":"#2b2b2b","color-h6":"#2b2b2b","color-page-heading-title":"#ffffff","color-page-heading-subtitle":"#ffffff","color-link":{"regular":"#333","hover":"#33C6C6","active":"#33C6C6"},"color-link-menu-top":{"regular":"#fff","hover":"#e6e6e6","active":"#e6e6e6"},"color-link-menu-middle":{"regular":"#2b2b2b","hover":"#33C6C6","active":"#33C6C6"},"color-link-menu-bottom":{"regular":"#fff","hover":"#e6e6e6","active":"#e6e6e6"},"color-link-menu-submenu":{"regular":"#fff","hover":"#fff","active":"#fff"},"color-menu-mobile-link":"#2b2b2b","color-header-top-content":"#ffffff","color-header-top":"#33C6C6","color-header-middle":"#ffffff","color-header-bottom":"#33C6C6","color-header-submenu":{"color":"#333","alpha":".95","rgba":"rgba(51,51,51,0.95)"},"color-header-submenu-hover":{"color":"#33C6C6","alpha":".99","rgba":"rgba(51,198,198,0.99)"},"color-footer-top-headings":"#ffffff","color-footer-top-text":"#d1d1d1","color-footer-top-link":{"regular":"#d1d1d1","hover":"#33C6C6","active":"#33C6C6"},"color-footer-top":"#212121","color-footer-top-border":{"color":"#333","alpha":".20","rgba":"rgba(51,51,51,0.2)"},"color-footer-bottom-headings":"#ffffff","color-footer-bottom-text":"#ffffff","color-footer-bottom-link":{"regular":"#ffffff","hover":"#33c6c6","active":"#33c6c6"},"color-footer-bottom":"#333333","color-element-portfolio-hover":{"color":"#33C6C6","alpha":".80","rgba":"rgba(51,198,198,0.8)"},"typo-body":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"16px","line-height":"26px"},"typo-h1":{"font-family":"Lato","font-options":"","google":"true","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"50px","color":""},"typo-h2":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"55px"},"typo-h3":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"24px","line-height":"30px"},"typo-h4":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"19px","line-height":"25px"},"typo-h5":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"17px","line-height":"24px"},"typo-h6":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"16px","line-height":"22px"},"typo-menu-top-middle":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"15px","line-height":"25px"},"typo-menu-top-top":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"13px","line-height":"26px"},"typo-menu-top-bottom":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"14px","line-height":"22px"},"typo-menu-submenu":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"14px","line-height":"24px"},"typo-page-heading-title":{"font-family":"Lato","font-options":"","google":"true","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"50px"},"typo-page-heading-subtitle":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"18px","line-height":"26px"},"page-heading-on-default":"1","page-heading-style-default":"heading-centered","page-heading-background-default":{"background-color":"#33c6c6","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"page-heading-spacing-default":{"padding-top":"50px","padding-bottom":"50px"},"page-comments-default":"1","page-padding":{"padding-top":"0","padding-bottom":"0"},"blog-listing-style":"blog-listing-card-3col","blog-listing-img":"full","blog-listing-meta-default":"1","blog-listing-meta-date-default":"1","blog-listing-meta-author-default":"1","blog-listing-meta-categories-default":"1","blog-listing-meta-comments-default":"1","blog-listing-meta-tags-default":"1","blog-single-meta-default":"1","blog-single-meta-date-default":"1","blog-single-meta-author-default":"1","blog-single-meta-categories-default":"1","blog-single-meta-comments-default":"1","blog-single-meta-tags-default":"1","n2mu-blog-full-post-content":"","portfolio-related-default":"1","portfolio-page-slug":"\/main\/portfolio-boxed-2-columns\/","layout-default":"0","layout-post-default":"1","layout-shop-default":"0","woo-catalog-mode-default":"","quick-translation":"","qt-404":"404","qt-404-title":"Ooops...","qt-404-subtitle":"We are sorry, but the page you are looking for does not exist.","qt-404-desc":"Please check entered address and try again or back to","qt-404-button":"Homepage","qt-single-portfolio-details-title":"Project details","qt-single-portfolio-details-client":"Client:","qt-single-portfolio-details-url":"Site URL:","qt-single-portfolio-details-date":"Date:","qt-single-portfolio-button-prev":"PREV","qt-single-portfolio-button-next":"NEXT","qt-single-portfolio-related":"Related Portfolio Items","qt-blog-listing-no-post":"Sorry, no posts matched your criteria.","qt-blog-listing-button":"Read more","qt-blog-no-comment":"No comments yet","qt-blog-one-comment":"1 comment","qt-blog-comments":"comments","qt-blog-comments-off":"Comments are Off","qt-blog-comment-password":"Enter your password to view comments.","qt-blog-tags":"Tags:","qt-comments-password-protected":"This post is password protected. Enter the password to view comments.","qt-comments-no-comments":"No Comments","qt-comments-one-comment":"One Comment","qt-comments-comments":"Comments","qt-comments-closed":"Comments are closed.","qt-comments-leave":"Leave A Comment","qt-comments-loggedin":"Logged in as","qt-comments-logout":"Log out","qt-comments-submit":"Comment","qt-comments-name":"Name","qt-comments-email":"Email","qt-comments-website":"Website","custom_css":"","redux-backup":"1"}

wp-content/themes/magnat/demo-import/onepage1/theme-options.txt
{"last_tab":"","favicon-img-default":{"url":"","id":"","height":"","width":"","thumbnail":""},"logo-option-default":"1","logo-img-default":{"url":"http:\/\/magnat.n2mu.studio\/onepage1\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-sticky-default":{"url":"http:\/\/magnat.n2mu.studio\/onepage1\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-mobile-default":{"url":"http:\/\/magnat.n2mu.studio\/onepage1\/wp-content\/themes\/magnat\/img\/logo.png","id":"","height":"","width":"","thumbnail":""},"logo-img-height-default":"28","logo-text-default":"Magnat","page-preloader":"","page-background-default":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"header-layout-default":"header1","header-top-height-default":"50","header-middle-height-default":"100","header-bottom-height-default":"50","header-mobile-height-default":"75","header-mobile-point":"1000","header-tools-search":"1","header-tools-cart":"1","header-sticky":"1","footer-columns-default":"4","footer-top-borders-default":"","footer-bottom-enabled-default":"1","copyrights":"Copyright \u00a9 2017 - <a href=\"http:\/\/magnat.n2mu.studio\" target=\"_blank\">Magnat Theme<\/a>","footer-top-spacing-default":{"padding-top":"50px","padding-bottom":"50px"},"footer-bottom-spacing-default":{"padding-top":"10px","padding-bottom":"10px"},"footer-background-default":{"background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"color-main-theme":"#33C6C6","color-buttons":"#33C6C6","color-buttons-font":"#ffffff","color-buttons-hover":"#33C6C6","color-buttons-font-hover":"#efefef","color-body":"#72787b","color-h1":"#2b2b2b","color-h2":"#2b2b2b","color-h3":"#2b2b2b","color-h4":"#2b2b2b","color-h5":"#2b2b2b","color-h6":"#2b2b2b","color-page-heading-title":"#ffffff","color-page-heading-subtitle":"#ffffff","color-link":{"regular":"#333","hover":"#33C6C6","active":"#33C6C6"},"color-link-menu-top":{"regular":"#fff","hover":"#e6e6e6","active":"#e6e6e6"},"color-link-menu-middle":{"regular":"#2b2b2b","hover":"#33C6C6","active":"#33C6C6"},"color-link-menu-bottom":{"regular":"#fff","hover":"#e6e6e6","active":"#e6e6e6"},"color-link-menu-submenu":{"regular":"#fff","hover":"#fff","active":"#fff"},"color-menu-mobile-link":"#2b2b2b","color-header-top-content":"#ffffff","color-header-top":"#33C6C6","color-header-middle":"#ffffff","color-header-bottom":"#33C6C6","color-header-submenu":{"color":"#333","alpha":".95","rgba":"rgba(51,51,51,0.95)"},"color-header-submenu-hover":{"color":"#33C6C6","alpha":".99","rgba":"rgba(51,198,198,0.99)"},"color-footer-top-headings":"#ffffff","color-footer-top-text":"#d1d1d1","color-footer-top-link":{"regular":"#d1d1d1","hover":"#33C6C6","active":"#33C6C6"},"color-footer-top":"#212121","color-footer-top-border":{"color":"#333","alpha":".20","rgba":"rgba(51,51,51,0.2)"},"color-footer-bottom-headings":"#ffffff","color-footer-bottom-text":"#ffffff","color-footer-bottom-link":{"regular":"#ffffff","hover":"#33c6c6","active":"#33c6c6"},"color-footer-bottom":"#333333","color-element-portfolio-hover":{"color":"#33C6C6","alpha":".80","rgba":"rgba(51,198,198,0.8)"},"typo-body":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"16px","line-height":"26px"},"typo-h1":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"50px","color":""},"typo-h2":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"55px"},"typo-h3":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"24px","line-height":"30px"},"typo-h4":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"19px","line-height":"25px"},"typo-h5":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"17px","line-height":"24px"},"typo-h6":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"16px","line-height":"22px"},"typo-menu-top-middle":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"15px","line-height":"25px"},"typo-menu-top-top":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"13px","line-height":"26px"},"typo-menu-top-bottom":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"14px","line-height":"22px"},"typo-menu-submenu":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"14px","line-height":"24px"},"typo-page-heading-title":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"45px","line-height":"50px"},"typo-page-heading-subtitle":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"","font-style":"","subsets":"latin","text-align":"","font-size":"18px","line-height":"26px"},"page-heading-on-default":"1","page-heading-style-default":"heading-centered","page-heading-background-default":{"background-color":"#33c6c6","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"page-heading-spacing-default":{"padding-top":"50px","padding-bottom":"50px"},"page-comments-default":"1","page-padding":{"padding-top":"0","padding-bottom":"0"},"blog-listing-style":"blog-listing-card-3col","blog-listing-img":"full","blog-listing-meta-default":"1","blog-listing-meta-date-default":"1","blog-listing-meta-author-default":"1","blog-listing-meta-categories-default":"1","blog-listing-meta-comments-default":"1","blog-listing-meta-tags-default":"1","blog-single-meta-default":"1","blog-single-meta-date-default":"1","blog-single-meta-author-default":"1","blog-single-meta-categories-default":"1","blog-single-meta-comments-default":"1","blog-single-meta-tags-default":"1","n2mu-blog-full-post-content":"","portfolio-related-default":"1","portfolio-page-slug":"\/main\/portfolio-boxed-2-columns\/","layout-default":"0","layout-post-default":"1","layout-shop-default":"0","woo-catalog-mode-default":"","quick-translation":"","qt-404":"404","qt-404-title":"Ooops...","qt-404-subtitle":"We are sorry, but the page you are looking for does not exist.","qt-404-desc":"Please check entered address and try again or back to","qt-404-button":"Homepage","qt-single-portfolio-details-title":"Project details","qt-single-portfolio-details-client":"Client:","qt-single-portfolio-details-url":"Site URL:","qt-single-portfolio-details-date":"Date:","qt-single-portfolio-button-prev":"PREV","qt-single-portfolio-button-next":"NEXT","qt-single-portfolio-related":"Related Portfolio Items","qt-blog-listing-no-post":"Sorry, no posts matched your criteria.","qt-blog-listing-button":"Read more","qt-blog-no-comment":"No comments yet","qt-blog-one-comment":"1 comment","qt-blog-comments":"comments","qt-blog-comments-off":"Comments are Off","qt-blog-comment-password":"Enter your password to view comments.","qt-blog-tags":"Tags:","qt-comments-password-protected":"This post is password protected. Enter the password to view comments.","qt-comments-no-comments":"No Comments","qt-comments-one-comment":"One Comment","qt-comments-comments":"Comments","qt-comments-closed":"Comments are closed.","qt-comments-leave":"Leave A Comment","qt-comments-loggedin":"Logged in as","qt-comments-logout":"Log out","qt-comments-submit":"Comment","qt-comments-name":"Name","qt-comments-email":"Email","qt-comments-website":"Website","custom_css":"","redux-backup":"1"}

wp-content/themes/magnat/framework/config/config.php
<?php defined( 'ABSPATH' ) OR die( 'This script cannot be accessed directly.' );

// Get header layout
function n2mu_default_config( $param ) {

    $config = array(
        'logo-option'                       =>  '1',
        'logo-img'                          =>  get_template_directory_uri() . '/img/logo.png',
        'logo-img-sticky'                   =>  get_template_directory_uri() . '/img/logo.png',
        'logo-img-mobile'                   =>  get_template_directory_uri() . '/img/logo.png',
        'logo-img-height'                   =>  '28',
        'logo-text'                         =>  'Magnat',
        'page-preloader'                    =>  false,
        'header-layout'                     =>  'header1',
        'header-top-height'                 =>  '50',
        'header-middle-height'              =>  '100',
        'header-bottom-height'              =>  '50',
        'header-mobile-height'              =>  '75',
        'header-mobile-point'               =>  '1000',
        'header-tools-search'               =>  true,
        'header-tools-cart'                 =>  true,
        'header-sticky'                     =>  false,
        'footer-columns'                    =>  '0',
        'footer-top-borders'                =>  false,
        'footer-bottom-enabled'             =>  true,
        'copyrights'                        =>  '<a href="http://magnat.n2mu.studio" target="_blank">Magnat theme</a>',
        'color-main-theme'                  =>  '#33C6C6',
        'color-buttons'                     =>  '#33C6C6',
        'color-buttons-font'                =>  '#fff',
        'color-buttons-hover'               =>  '#33C6C6',
        'color-buttons-font-hover'          =>  '#efefef',
        'color-body'                        =>  '#72787b',
        'color-h1'                          =>  '#2b2b2b',
        'color-h2'                          =>  '#2b2b2b',
        'color-h3'                          =>  '#2b2b2b',
        'color-h4'                          =>  '#2b2b2b',
        'color-h5'                          =>  '#2b2b2b',
        'color-h6'                          =>  '#2b2b2b',
        'color-page-heading-title'          =>  '#2b2b2b',
        'color-page-heading-subtitle'       =>  '#2b2b2b',
        'color-link-regular'                =>  '#333',
        'color-link-hover'                  =>  '#33C6C6',
        'color-link-active'                 =>  '#33C6C6',
        'color-link-menu-top-regular'       =>  '#fff',
        'color-link-menu-top-hover'         =>  '#e6e6e6',
        'color-link-menu-top-active'        =>  '#e6e6e6',
        'color-link-menu-middle-regular'    =>  '#2b2b2b',
        'color-link-menu-middle-hover'      =>  '#33C6C6',
        'color-link-menu-middle-active'     =>  '#33C6C6',
        'color-link-menu-bottom-regular'    =>  '#fff',
        'color-link-menu-bottom-hover'      =>  '#e6e6e6',
        'color-link-menu-bottom-active'     =>  '#e6e6e6',
        'color-link-menu-submenu-regular'   =>  '#fff',
        'color-link-menu-submenu-hover'     =>  '#fff',
        'color-link-menu-submenu-active'    =>  '#fff',
        'color-menu-mobile-link'            =>  '#2b2b2b',
        'color-header-top-content'          =>  '#fff',
        'color-header-top'                  =>  '#33C6C6',
        'color-header-middle'               =>  '#fff',
        'color-header-bottom'               =>  '#33C6C6',
        'color-header-submenu-color'        =>  '#333',
        'color-header-submenu-alpha'        =>  '.95',
        'color-header-submenu-hover-color'  =>  '#33C6C6',
        'color-header-submenu-hover-alpha'  =>  '.99',
        'color-footer-top-headings'         =>  '#fff',
        'color-footer-top-text'             =>  '#d1d1d1',
        'color-footer-top-link-regular'     =>  '#d1d1d1',
        'color-footer-top-link-hover'       =>  '#33C6C6',
        'color-footer-top-link-active'      =>  '#33C6C6',
        'color-footer-top'                  =>  '#212121',
        'color-footer-top-border-color'     =>  '#333',
        'color-footer-top-border-alpha'     =>  '.20',
        'color-footer-bottom-headings'      =>  '#fff',
        'color-footer-bottom-text'          =>  '#d1d1d1',
        'color-footer-bottom-link-regular'  =>  '#d1d1d1',
        'color-footer-bottom-link-hover'    =>  '#33C6C6',
        'color-footer-bottom-link-active'   =>  '#33C6C6',
        'color-footer-bottom'               =>  '#111111',
        'color-element-portfolio-hover-color' =>  '#33C6C6',
        'color-element-portfolio-hover-alpha' =>  '.80',
        // Typography - TODO
        'page-heading-on'                   =>  true,
        'page-heading-style'                =>  'heading-centered',
        'page-heading-background'           =>  '#efefef',
        'page-heading-spacing-top'          =>  '70px',
        'page-heading-spacing-bottom'       =>  '70px',
        'page-padding-top'                  =>  '30px',
        'page-padding-bottom'               =>  '30px',
        'page-comments'                     =>  true,
        'blog-listing-style'                =>  'blog-listing-simple',
        'blog-listing-img'                  =>  'n2mu-cropped-big',
        'blog-listing-meta'                 =>  true,
        'blog-listing-meta-date'            =>  true,
        'blog-listing-meta-author'          =>  true,
        'blog-listing-meta-categories'      =>  true,
        'blog-listing-meta-comments'        =>  true,
        'blog-listing-meta-tags'            =>  true,
        'blog-single-meta'                  =>  true,
        'blog-single-meta-date'             =>  true,
        'blog-single-meta-author'           =>  true,
        'blog-single-meta-categories'       =>  true,
        'blog-single-meta-comments'         =>  true,
        'blog-single-meta-tags'             =>  true,
        'n2mu-blog-full-post-content'       =>  false,
        'portfolio-related'                 =>  false,
        'portfolio-page-slug'               =>  '',
        'layout'                            =>  '0',
        'layout-post'                       =>  '1',
        'layout-shop'                       =>  '0',
        'woo-catalog-mode'                  =>  false,
        'quick-translation'                 =>  false,
    );

    if (array_key_exists($param, $config)) {
        return $config[$param];
    } else {
        return '';
    }
    
    
}